### PR TITLE
Moves pre-round observers to the wizden

### PR DESCRIPTION
**Pull request by lewcc**
## What Does This PR Do
Moves roundstart observers to the wizard's den, spawning them on the chairs. This specifically only impacts folks who chose to observe before the round starts.

## Why It's Good For The Game
This is just some quick QoL, and kind of formalizes a bit of a tradition for observer mains haha

## Testing
Observed before the round started, was moved to the wizden.

## Changelog
:cl:
add: If you observe before roundstart, you'll now spawn in the wizard's den.
/:cl:

### CODE REVIEW

File: code/game/objects/effects/landmarks.dm [hunk 1]
```diff
@@ -227,6 +227,16 @@ INITIALIZE_IMMEDIATE(/obj/effect/landmark/newplayer_start) //Without this you sp
...
+/obj/effect/landmark/spawner/roundstart_observer
+	name = "Roundstart Observer"
+	icon_state = "spooky"
+
+/obj/effect/landmark/spawner/roundstart_observer/Initialize(mapload)
+	spawner_list = GLOB.roundstart_observer_start
+	return ..()
+
+
+
```
Comment (GDNgit) [inline code/game/objects/effects/landmarks.dm:239]:
```suggestion

```

File: code/modules/mob/new_player/new_player.dm [hunk 2]
```diff
@@ -185,9 +185,14 @@
 				to_chat(src, "<span class='notice'>As you observed [period_human_readable], you can freely toggle antag-hud without losing respawnability.</span>")
 			observer.started_as_observer = 1
 			close_spawn_windows()
-			var/obj/O = locate("landmark*Observer-Start")
+			var/obj/spawn_point
+			if(SSticker.current_state < GAME_STATE_PLAYING)
+				spawn_point = pick(GLOB.roundstart_observer_start)
+			else
+				spawn_point = locate("landmark*Observer-Start")
```
Comment (GDNgit) [inline code/modules/mob/new_player/new_player.dm:192]: God we need to obliterate all the tag usage soon

**Review by GDNgit — CHANGES_REQUESTED** [review]

File: code/modules/mob/new_player/new_player.dm (hunk 2 above)
Comment (lewcc) [reply code/modules/mob/new_player/new_player.dm:192]: yeah I wasn't entirely sure what was going on here and I don't quite like it

File: code/game/objects/effects/landmarks.dm [hunk 3]
```diff
@@ -227,6 +227,13 @@ INITIALIZE_IMMEDIATE(/obj/effect/landmark/newplayer_start) //Without this you sp
 /obj/effect/landmark/spawner/nuke_code
 	name = "nukecode"
 
+/obj/effect/landmark/spawner/roundstart_observer
+	name = "Roundstart Observer"
+	icon_state = "spooky"
+
+/obj/effect/landmark/spawner/roundstart_observer/Initialize(mapload)
+	spawner_list = GLOB.roundstart_observer_start
+	return ..()
```
Comment (GDNgit) [inline code/game/objects/effects/landmarks.dm:236]:
```suggestion
	return ..()
	
```
odd github behavior

**Review by DGamerL — APPROVED** [review]

**Pull request merged**